### DISCUSSION TITLE
Durée de validité cookies ?

### DISCUSSION THREAD

**Discussion by Delit-dinfluence** [AmauriC/tarteaucitron.js#1238] [Q&A] [answered]
Bonjour à tous,
J'aimerais configurer tarteaucitron pour garder les cookies valides 6 mois.
Je n'ai pas trouvé de solution qui fonctionne pour le moment...
Dois-je avoir une version payante ou existe t-il une option avec la version gratuite ?
Merci !!!

**Comment by jipexu** [comment]
Hi
probably change this variable     timeExpire = 31536000000, in tarteaucitron.js

**Comment by AmauriC** [comment, marked as answer]
Il faut définir la variable ci-dessous avant le chargement de tarteaucitron:

```
<script>
var tarteaucitronForceExpire = 180; // 30 jours * 6 mois
</script>
```

**Reply by Delit-dinfluence** [reply]
Merci beaucoup.
